draft-packages/empty-state/package.json

```diff
@@ -35,11 +35,12 @@
     "@kaizen/component-base": "^1.1.0",
     "@kaizen/component-library": "^15.0.3",
     "@kaizen/draft-illustration": "^5.1.13",
-    "@types/classnames": "^2.3.0",
+    "@kaizen/typography": "^2.2.3",
     "classnames": "^2.3.1"
   },
   "devDependencies": {
-    "rimraf": "^3.0.2"
+    "rimraf": "^3.0.2",
+    "@types/classnames": "^2.3.0"
```

Comment:
Nice catch moving that to `devDependencies` 👏

Reply:
I actually think it can probably be removed since it's in the root package.json 😬 Unless something explodes which proves that it must be within every package

Reply:
Good point 🤔 will try it out and see if anything explodes